setup.py

```diff
@@ -78,7 +78,8 @@
         "protobuf>=3.0.0,<4",
         "pypiwin32>=223;platform_system=='Windows'",
         "requests>=2.16.0,<3.0.0",
-        "websockets>=7.0.0,<8.0.0",
+        "websockets>=8.1.0,<9.0.0",
+        "cytoolz>=0.10.1",
```

Comment:
need to get rid of this once python3.8 support for `eth-utils` is released.

Reply:
Available in `v1.8.0` now.